app/build.gradle

```diff
@@ -34,3 +34,4 @@ dependencies {
     androidTestImplementation "com.android.support.test:runner:$rootProject.testRunnerRulesVersion"
     androidTestImplementation "com.android.support.test.espresso:espresso-core:$rootProject.espressoVersion"
 }
+
```

Comment:
Unnecessary. Pls remove

Reply:
> You can open branches as much as it's necessary. Do some study on git branching. It seems you lack understanding about git version controlling. You don't have to rely on the same branch. Different features can be implemented in different branches. Makes production easier and faster.

@CloudyPadmal Yes, I have been brushing up on my VCS skills lately. And thanks a lot for the suggestion sir.

Reply:
@GOVINDDIXIT I understand the idea of removing espresso as it is only available for Android 8.0 and above. But runner is necessary for running tests and it facilitates loading the test package and the app under test onto a device or emulator, running your tests, and reporting the results. Moreover these are the basic default dependencies created by Android Studio itself. Is there any concrete reason on why to remove this.

Reply:
I am asking to remove extra line

Reply:
@GOVINDDIXIT Okay cool sure. I got it.

Reply:
@GOVINDDIXIT is it a problem if I remove the extra line in the next PR. Because the changes in this PR have already been approved and so I don't want to make any further changes in this one. Is it cool?

Reply:
@GOVINDDIXIT I removed the extra line in #34 . Thank you.

Reply:
> @GOVINDDIXIT I removed the extra line in #34 . Thank you.

This is bad practice. You can always make change in an approved PR, If something got missed

Reply:
Okay cool will keep that in mind @GOVINDDIXIT . Thanks.